dll/win32/browseui/explorerband.cpp

```diff
@@ -848,6 +916,7 @@ BOOL CExplorerBand::NavigateToCurrentFolder()
     /* find PIDL into our explorer */
     result = NavigateToPIDL(explorerPidl, &dummy, TRUE, FALSE, TRUE);
     m_bNavigating = FALSE;
+    ILFree(explorerPidl);
```

Comment:
Is this one the memory leak that you mentioned in the PRs description? Or is there another one somewhere else that I overlooked?

Reply:
Yes, this is the leak